eras/alonzo/impl/src/Cardano/Ledger/Alonzo/Rules/Utxow.hs

```diff
@@ -76,15 +76,9 @@ import Cardano.Ledger.Shelley.Delegation.Certificates
   )
 import Cardano.Ledger.Shelley.LedgerState
   ( UTxOState (..),
-    propWits,
     witsFromTxWitnesses,
   )
-import Cardano.Ledger.Shelley.Rules
-  ( ShelleyUtxowEvent (UtxoEvent),
-    ShelleyUtxowPredFailure (..),
-    UtxoEnv (..),
-    validateNeededWitnesses,
-  )
+import Cardano.Ledger.Shelley.Rules (ShelleyUtxowEvent (UtxoEvent), ShelleyUtxowPredFailure (..), UtxoEnv (..), propWits, validateNeededWitnesses)
```

Comment:
sorry for the bike shedding, but I prefer using the vertical space instead, it makes for better diff history. that's just my opinion, though, and if everyone else disagrees we can use the horizontal space.

Reply:
For lines that are that long, I do agree.

Reply:
I thought ormolise did that automatically.  I changed it.